package.json

```diff
@@ -24,6 +24,7 @@
     "babel-preset-babili": "^0.1.4",
     "babel-preset-es2015": "^6.24.1",
     "gulp": "^3.9.1",
+    "js-test-buddy": "^0.0.3",
```

Comment:
Is this something anyone can extend (or will need to extend), or do you imagine it not being needed to?

Reply:
Likely won't need to change very often. It's something I built so we can submit a PR to make updates, or even fork in the future if we need to.

Reply:
Yeh, as long as there’s a way to update if you go away on holiday or get hit by a bus I’m happy (not about you being hit by a bus in that scenario obviously… 😱 )

Reply:
I've added you, Simon, and Nick as collaborators on the projects.